.travis.yml

```diff
@@ -38,6 +42,8 @@ script:
     - cppcheck --force --enable=warning,missingInclude --error-exitcode=1 . >build.log 2>&1 || (cat build.log && exit 1)
     - mkdir image
     - tar zxf download-cache/openssl-${OPENSSL_VERSION}.tar.gz && cd openssl-${OPENSSL_VERSION}
+    - if [ "$CHOST" == "i686-w64-mingw32" ]; then export m=mingw; fi
```

Comment:
As $m is later used as the target spec for openssl compilation, "target=mingw" etc instead of "m=mingw" would be easier to understand for others looking a this. Just nitpicking :)